google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -336,6 +336,9 @@ clouddebug.runconfig.formoredetails=For more details,
 clouddebug.runconfig.documentation.url=https://cloud.google.com/debugger/
 clouddebug.runconfig.documentation.url.text=click here
 
+gcs.panel.toolbar.refresh.hover.text=Refresh
```

Comment:
note to self for future implementation: probably already implied but to be explicit - when implement a listener like this, we'd wouldn't automatically refresh the list. Instead we'd have some text at the top, for example:

`Bucket changes detected. Click here to refresh the list.`